Makefile

```diff
@@ -5,3 +5,4 @@ clean  :; dapp clean
 test   :; ./test-dssspell.sh $(match)
 deploy :; make && dapp create DssSpell | xargs ./verify.py DssSpell
 flatten :; hevm flatten --source-file "src/Goerli-DssSpell.sol" > out/flat.sol
+archive-spell :; ./archive-spell.sh $(name)
```

Comment:
I would name it just archive, to keep it short.

Reply:
i tried that and got `make: `archive' is up to date.` returned which made me think that `make archive` is a reserved command.

Reply:
Oh! right.